src/routes/ViewAllLogs/ViewAllLogs.js

```diff
@@ -411,13 +408,19 @@ class ViewAllLogs extends Component {
     } = this.props;
     const { isLogsDeletionInProgress } = this.state;
 
+    let isAllSelectedByValues = isAllSelected;
```

Comment:
@jakhongirabdukhamdiov what does `isAllSelectedByValues` mean? Could you come up more meaningful name?

Reply:
in the checkboxlist component you can see that the selected records are stored in session storage and decides isAllSelected by the length of the values that are stored and current records of the new page. So when we change from one pagination page to the next it shows it's all selected if next page also has the same number of records, not considering the values that are stored, that's the main reason I chose the name of byValues.